### PR TITLE
[Xedra Evolved] Magic MTFs

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
Mods "Added few magic related profs"
#### Purpose of change
Wanted to play as a dreamforger from the start, found lack of such option, made this
#### Testing
Seems working for me
#### Additional context
I'd like to add some recipes for forger and inventor to know from the start, but profs can't make this, directly or using EoC is not possible. TBD, i guess

### CODE REVIEW

File: data/mods/Xedra_Evolved/player/professions.json [hunk 1]
```diff
@@ -263,6 +262,190 @@
...
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "sports_bra", "boxer_shorts" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "xe_eater",
+    "name": "Mobile Taks Force: Upsilon 4 \"Devourers\"",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/player/professions.json:279]:
```suggestion
    "name": "Mobile Task Force: Upsilon 4 \"Devourers\"",
```

File: data/mods/Xedra_Evolved/player/professions.json [hunk 2]
```diff
@@ -263,6 +262,190 @@
...
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "sports_bra", "boxer_shorts" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "xe_dreamer",
+    "name": "Mobile Taks Force: Theta 1 \"Middle-Management\"",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/player/professions.json:334]:
```suggestion
    "name": "Mobile Task Force: Theta 1 \"Middle-Management\"",
```

File: data/mods/Xedra_Evolved/player/professions.json [hunk 3]
```diff
@@ -263,6 +262,190 @@
...
+    "description": "Theta 1, also known as \"kindergarten\" between other task forces, is used when missions are expected to be too dangerous to send people or when reconnaissance can't obtain any important information about the target, which is why otherworldly creatures, summoned by dreamers, are sent.  It's not the best course of action, but the unit's nearly zero fatality rate makes it worth using.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/player/professions.json:335]: >It's not the best course of action, but the unit's nearly zero fatality rate makes it worth using.

So why is it not the best course of action, collateral damage? 🤔

File: data/mods/Xedra_Evolved/player/professions.json [hunk 4]
```diff
@@ -263,6 +262,190 @@
...
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "sports_bra", "boxer_shorts" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "xe_inventor",
+    "name": "Mobile Taks Force: Nu 3 \"Techno Savvy\"",
+    "description": "Despite the \"Task Force\" in their name, Nu 3 is rarely called into the actual mission; their main task is reconnaissance and developing tools to support another task force, even if these tools last only a few weeks.  But when they do accept a task, it's almost guaranteed to be successful.  It should've been your first day here, but something went really wrong.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/player/professions.json:381]:
```suggestion
    "name": "Mobile Task Force: Nu 3 \"Techno Savvy\"",
    "description": "Despite the \"Task Force\" in their name, Nu 3 is rarely called into the actual mission; their main task is reconnaissance and developing tools to support another task force, even if these tools last only a few weeks.  But when they do accept a task, it's almost guaranteed to be successful.  It should've been your first day here, but something went really wrong.",
```

File: data/mods/Xedra_Evolved/player/professions.json [hunk 5]
```diff
@@ -263,6 +262,190 @@
...
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "sports_bra", "boxer_shorts" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "xe_dreamsmith",
+    "name": "Mobile Taks Force: Tau 8 \"Anvil\"",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/player/professions.json:424]:
```suggestion
    "name": "Mobile Task Force: Tau 8 \"Anvil\"",
```

**Review by TheShadowFerret — CHANGES_REQUESTED** [review]
~~What's a taks~~

File: data/mods/Xedra_Evolved/player/professions.json [hunk 6]
```diff
@@ -263,6 +262,190 @@
...
+      { "level": 3, "name": "electronics" },
+      { "level": 4, "name": "deduction" }
+    ],
+    "spells": [ { "id": "firmament_driver", "level": 7 }, { "id": "cable_whip", "level": 7 } ],
+    "traits": [ "INVENTOR" ],
+    "proficiencies": [ "prof_metalworking", "prof_welding_basic", "prof_elec_soldering" ],
+    "items": {
+      "both": {
+        "ammo": 100,
+        "items": [
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/player/professions.json:397]: You can create a recipe book item that only spawns with this profession and has whichever Inventor recipes you want the class to start with.

File: data/mods/Xedra_Evolved/player/professions.json (hunk 3 above)
Comment (GuardianDll) [reply data/mods/Xedra_Evolved/player/professions.json:335]: Their intellectual abilities do not match their strength, that's why the only thing summoner can do is point them into direction (which is also non-trivial task) and command to destroy everything, yeah

File: data/mods/Xedra_Evolved/player/professions.json (hunk 6 above)
Comment (GuardianDll) [reply data/mods/Xedra_Evolved/player/professions.json:397]: But it won't be counted into inventor/dreamsmith recipe knowledge, and when they try to research more, they effectively obtain it twice (first in the book, and secondnby actual research)

**Pull request merged**